vulcan.yaml

```diff
@@ -25,6 +25,29 @@ conf:
   # Number of checks to run concurrently
   concurrency: 5
 
+policies:
+  - name: default
+    checks: 
```

Comment:
What if we create a new file vulcan-policies.yaml inside /script and we reference this file in the script.
```sh
./vulcan-local -c vulcan.yaml -c script/vulcan-policies.yaml ....
```

Also I would expand the README.md Polices with an example, ... and perhaps an explanation using the remote file to see the full power

```sh
VULCAN_LOCAL_CONFIG=https://raw.githubusercontent.com/adevinta/vulcan-local/master/script/vulcan-policies.yaml

./vulcan-local -c vulcan.yaml -p lightweight
```